docs/diagnostics/PX1007.md

```diff
@@ -19,7 +19,12 @@ Any DAC or DAC extension and their public property fields should have one of the
 The descriptions of the DACs, DAC extensions, and their members can be displayed in the DAC Schema Browser and used for building the API and DAC references for the Acumatica ERP libraries.
 
 > **Note:**
-> For the mapped DAC fields, the original mapped property is determined from the `BqlField` or `BqlTable` values specified in attributes of the projection DAC property. Non-mapped properties (unbound or calculated at runtime) are processed as normal DAC properties.
+> For the mapped DAC fields, the original mapped property is determined using one of the following ways:
+> * If the field has an attribute with a `BqlField` property, and this property's value is mapped to a BQL field of an original projection DAC, this BQL field is used for mapping
+> * If the field has an attribute with a `BqlTable` property, and this property's value is the name of the type of the projection DAC, this BQL field is used for mapping
+>   The system seaches the mapped field in the specified DAC by its name, so the name of the mapped DAC field should coincide with name of the the projection DAC field.
+> * If no `BqlField` or `BqlTable` property is available, the system searches for the mapped field among the fields with the same names in the base DACs. The system takes the first such found field.
```

Comment:
@EAndrosova  I think the documentation should specify that declaring both `BqlField` and `BqlTable` property is not supported by the system.